libs/common/domain/src/lib/platform.service.interface.ts

```diff
@@ -0,0 +1,17 @@
+import { Observable } from 'rxjs'
+import { UserModel } from './model/user/user.model'
+import { Organization } from './model/record/organization.model'
+
+export abstract class PlatformServiceInterface {
+  abstract getTye(): string
+  abstract getApiVersion(): Observable<string>
+  abstract isApiCompatible(): Observable<boolean>
```

Comment:
I'm wondering if this is really needed? In what scenario could we possibly want to still run the application with a non-compatible backend? Maybe we should just throw a "incompatible backend" error and stop running the app

Reply:
I'll treat that in the next PR where I will effectively test for the compatibility